src/app/pages/dashboard/components/test-centre-journal-card/test-centre-journal-card.html

```diff
@@ -1,12 +1,12 @@
 <button class="test-centre-journal-button" (click)="navigateToTestCentreJournal()">
-  <ion-card class="test-centre-journal-card">
+  <ion-card class="test-centre-journal-card" id="test-centre-journal-card">
     <div id="card-content-wrapper">
       <ion-grid>
         <ion-row class="icon-row">
           <img src="assets/imgs/dashboard/go-to-test-centre-journal.png" alt="TestCentreJournal" class="icon-test-centre"/>
         </ion-row>
         <ion-row>
-          <ion-button mode="ios" class="mes-primary-blue-button">
+          <ion-button id="test-centre-journal" mode="ios" class="mes-primary-blue-button">
```

Comment:
same again, call this "test-centre-journal-button"

Reply:
done